src/components/toasts/AddToCartToast.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+
+import { toast } from "sonner";
+import { useRouter } from "next/navigation";
+import Image from "next/image";
+import { formatCurrency } from "@/lib/formatters";
+import { Button } from "@/components/ui/button";
+
+type AddToCartToastProps = {
+  productName: string;
+  price: number;
+  imagePath: string;
+};
+
+const AddToCartToast = ({
+  productName,
+  price,
+  imagePath,
+}: AddToCartToastProps) => {
+  const router = useRouter();
```

Comment:
Doesn't seem like you used router in this file. I think it's safe to delete it and its import on Line 4

Reply:
used on line 45